src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -116,6 +116,17 @@ Copyright (c) .NET Foundation. All rights reserved.
   
   <!-- Default item includes (globs and implicit references) -->
   <Import Project="Microsoft.NET.Sdk.DefaultItems.props" />
+
+  <!-- Put workload resolution behind a feature flag.  It can be enabled either by setting the MSBuildEnableWorkloadResolver environment variable to true, or by
+       putting an EnableWorkloadResolver.sentinel file beside the MSBuild SDK resolver DLL -->
+  <PropertyGroup Condition="'$(MSBuildEnableWorkloadResolver)' == ''">
+    <__EnableWorkloadResolverSentinelPath Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildBinPath)\EnableWorkloadResolver.sentinel</__EnableWorkloadResolverSentinelPath>
```

Comment:
Tested the feature flag works for on/off * commandline/VS * environment/sentinel, 8 scenarios?